www/activities/settings/js/settings.js

```diff
@@ -792,6 +793,11 @@ app.Settings = {
         settings.statistics["last-stat"] = lastStat;
       }
       window.localStorage.removeItem("last-stat");
+      
+      // Show Brands setting should be initialized to "true", as is the historical default behaviour 
+      if (settings.diary["show-brands"] === undefined) {
+        settings.diary["show-brands"] = true;
+      }
```

Comment:
@hellsend This looks good. However, I forgot to mention that you also need to increment the `currentSettingsSchemaVersion` variable defined at the top of this file. Otherwise, the settings migration won't actually be applied.

Reply:
Oh, interesting, lemme get on this!

Reply:
Done. Thanks for pointing this out, it clearly had escaped me.